data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -59,22 +66,12 @@ def __init__(self):
         )
         self.CONTAINS_AGRIVALUE = "Contains agricultural value"
 
-        self.COLUMNS_TO_KEEP = [
```

Comment:
This logic is all replaced by the YAML config now, right?

Reply:
Yes!